src/Bridge/Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -186,6 +222,13 @@ protected function filterProperty(string $property, $value, QueryBuilder $queryB
      */
     protected function addWhereByStrategy(string $strategy, QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator, string $alias, string $field, $value, bool $caseSensitive)
     {
+        // check if we have metadata
+        if (($metadata = func_get_arg(7)) instanceof ClassMetadata) {
```

Comment:
you should add this new parameter as a comment in the method signature too

Reply:
fixed (but I would like to make sure I handled it correctly)